doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_architecture_reset.rst

```diff
@@ -0,0 +1,62 @@
+.. _ug_nrf54h20_architecture_reset:
+
+nRF54H20 reset behavior
+#######################
+
+.. contents::
+   :local:
+   :depth: 2
+
+.. TODO
+
+nrfutil allows for different types of reset behavior on the various cores of the nRF54H20 SoC, based on the current lifecycle state of the device.
```

Comment:
Before the section about nrfutil we should add a section about reset handling in the system. Briefly, we should capture that:

1. Reset of Secure Domain resets the SoC
2. Reset of System Controller resets the SoC
3. Reset of Application CPU causes reset of:
    * all peripherals (and VPRs) owned by Application
    * Radio CPU
    * all peripherals (and VPRs) owned by Radio
4. Reset of Radio CPU causes reset of:
    * all peripherals (and VPRs) owned by Radio
    * Application CPU
    * all peripherals (and VPRs) owned by Application